### ISSUE TITLE
[Feature Request] Allow creating a special type of playlist that is auto generated from the most recent X number of videos from a group of creators/channels.

### ISSUE THREAD

**Issue by LorneCash** [futo-org/grayjay-android#371]
I often listen to YouTube while driving and I would love to be able to use my steering wheel controls to control next track and previous track when listening to my subscriptions tab.

The next track/previous track works in playlists but not on the subscriptions tab.  So it would make sense to me to implement this as a special type of playlist that is auto generated from the subscriptions and then just set a maximum number of items in the playlist.  A practical limit for me personally would be 25 but I may want more if I'm going on a longer road trip.

As a first iteration, maybe just support all subscriptions, but I really think adding a filter for specific creator/channels should be the ultimate goal.

I would absolutely love to be able to consume news content in this way!

**Comment by stefancruz** [comment]
Newpipe has a feature which allows to create group of channels. Then you can see the vídeos for the selected group.

After this, in terms of ui, the only thing missing is a "play all" or "queue all" button.

What do you thing @LorneCash?

**Comment by LorneCash** [comment]
I'm going to refer to my suggestion as dynamic playlists and the other as channel groups.

I like the idea for a channel groups and i would definitely use them if dynamic playlists weren't an option but from a usability standpoint if I had both channel groups and dynamic playlists I don't imagine myself ever using the channel group because the dynamic playlist would have every benefit of a channel group and also the ability to use next track previous track from my earbuds or steering wheel controls.

Can you think of any feature that a channel group would provide that a dynamic playlist wouldn't?

**Comment by defiling9046** [comment]
@Zvonimir-FUTO do you think adding the "play entire feed" option to the subscriptions feed **(which implies it would also work with subscription group feeds)** would be the best alternative solution to this similar to how you can already do this via a feed generated by a search query as shown below:

Reproduction steps to do it using a search query for reference:

1. Search something
2. Click on the "options" button on one of the videos listed
3. Click "play entire feed"
4. (OPTIONAL) if the user wants to save it as a playlist they can view the queue and then save it as a playlist

<img width="1080" height="2424" alt="Image" src="https://github.com/user-attachments/assets/b54b81fa-5c42-4512-9320-80398d97bf0b" />
<img width="1080" height="2424" alt="Image" src="https://github.com/user-attachments/assets/18ed93b2-55c2-47db-a24b-e027c175ba34" />
<img width="1080" height="2424" alt="Image" src="https://github.com/user-attachments/assets/4ca8ff33-1790-45f5-92e7-a27cc11bd454" />
<img width="1080" height="2424" alt="Image" src="https://github.com/user-attachments/assets/d79f864a-d058-4361-bda8-afcc5350ffd6" />
<img width="1080" height="2424" alt="Image" src="https://github.com/user-attachments/assets/461c6188-6b87-4a9c-9f07-35558379398c" />

**Comment by Zvonimir-FUTO** [comment]
Well this could work as a workaround by creating a Subscription group, selecting it and clicking on Options button under any video and tapping in **Play Feed as Queue** but you would have to this each time. 

Saving it as a playlist wouldn't work dynamically. It would just save current queue/feed as a playlist instead of always updating with fresh videos from same creators.

**Comment by defiling9046** [comment]
> Well this could work as a workaround by creating a Subscription group, selecting it and clicking on Options button under any video and tapping in **Play Feed as Queue** but you would have to this each time.
> 
> Saving it as a playlist wouldn't work dynamically. It would just save current queue/feed as a playlist instead of always updating with fresh videos from same creators.

Right, that makes sense. Apologies, the dynamic part of the feature request skipped my head.
